src/Cake.Common/Tools/OctopusDeploy/PushPackageArgumentBuilder.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.OctopusDeploy
+{
+    internal class PushPackageArgumentBuilder : OctopusDeployArgumentBuilder<PushPackageSettings>
```

Comment:
I might be missing something, but isn't that exactly what `OctopusDeployArgumentBuilder` is? A common base class? Is this just a naming thing?